### DISCUSSION TITLE
Testing Discussions

### DISCUSSION THREAD

**Discussion by shannonrankin** [SAEL-SWFSC/PAM-SI-Mtg-March2024#3] [Announcements]
How well does this format work for taking meeting notes?

**Comment by rebecca-vanhoeck** [comment]
I think this will be more for conversation?

**Reply by shannonrankin** [reply]
Perhaps both-- Conversation & meeting notes

**Reply by shannonrankin** [reply]
This is too distracting for meeting notes-- should stick with Google Docs

**Comment by shannonrankin** [comment]
Question: can we export this as a document for review/copy/cut/paste?
